doc/database-backup-and-restore.md

```diff
@@ -39,34 +39,31 @@ You will need to be a user of the services GPaaS account. You're OK if you can s
     cf services # to get the SERVICE_NAME
     cf conduit SERVICE_NAME -- pg_dump --file STAGING_DATA_FILE_NAME.sql
     ```
-1. Find the Database role name from the database dump. It should be of the form: `rdsbroker_<uuid>_manager`
+1. Destroy the existing database
     ```
-    grep -n rdsbroker PROD_DATA_FILE_NAME.sql
+    cf unbind-service beis-roda-pentest beis-roda-pentest-postgres
+    cf unbind-service beis-roda-pentest-worker beis-roda-pentest-postgres
+    cf delete-service beis-roda-pentest-postgres
     ```
-1. Connect to the target database with a PostgreSQL prompt
+1. Use Terraform to recreate it (this may take 10-20 minutes)
     ```
-    cf target -s staging
-    cf services # to get the SERVICE_NAME for the destination postgres
-    cf conduit SERVICE_NAME -- psql
+    terraform apply --var-file=pentest.tfvars --var 'data_migrate=false'
```

Comment:
Instead of documenting steps to update the tfvars correctly I've included a deliberate `var` flag on the end of this command to ensure for any deploys after a data wipe it doesn't migrate the data.